stdlib/public/core/UnsafeRawPointer.swift

```diff
@@ -455,12 +455,27 @@ public struct UnsafeRawPointer: _Pointer {
   /// - Returns: A new instance of type `T`, read from the raw bytes at
   ///   `offset`. The returned instance isn't associated
   ///   with the value in the range of memory referenced by this pointer.
+  @inlinable
```

Comment:
Do we want this to be always-inline (or even transparent?)? I feel like that might be appropriate for a single load.

Reply:
It's already `@_alwaysEmitIntoClient` which does those things, though I like to prefix that with `@inlinable` because that is meaningful to more people.

Reply:
aEIC doesn't force inlining into callsites, no. Just into the binary. We actually have a bunch of inline(never) aEIC code in the stdlib, which means "emit exactly one copy into the client"